### DISCUSSION TITLE
jsconfig.json intellisense doesn't apply to jupyter notebook code cells

### DISCUSSION THREAD

**Discussion by c3-WesleyPickering** [microsoft/vscode-discussions#176] [Extension Development QnA] [answered]
I am currently looking for a way to modify the JS/TS language server being used in VS Code Jupyter notebook's code cells. In my project I created a jsconfig.json file and included paths to .ipynb and .js files. My intention is to be able to see the same autocomplete and GoTo functionality for the same type in both types of files, however, it currently only works in .js files, whereas the .ipynb files are limited to base js/ts language server intellisense.

Screenshot showing autocomplete for .js file -- UiTestType1 shows up since it is defined in UiTestType1.d.ts. 
![js intellisense](https://user-images.githubusercontent.com/60113255/195960243-99cb76b0-ec41-4117-aae5-86b6bbf36157.png)

I would expect the same autocomplete to appear for .ipynb files such as in screenshot below 
![ipynb intellisense](https://user-images.githubusercontent.com/60113255/195960312-349f09c5-5a93-4125-8367-89ffac3849a7.png)

Attaching the project I used for screenshots
[testPkg.zip](https://github.com/microsoft/vscode-discussions/files/9790792/testPkg.zip)

**Comment by isidorn** [comment]
@rebornix might be able to help here.

**Reply by rebornix** [reply]
The typescript language service doesn't seem to handle notebooks yet @mjbvz

**Comment by mjbvz** [comment, marked as answer]
This expected today and falls under https://github.com/microsoft/vscode/issues/129482

**Reply by c3-WesleyPickering** [reply]
@mjbvz Is this expected to work now? The linked ticket is still `open`

**Reply by tonescones** [reply]
@mjbvz https://github.com/microsoft/vscode/issues/129482 doesn't seem related to this
